### PR TITLE
Remove standalone activities, change section structure

### LINKED ISSUE #102: Do we want separately dereferencable individual Activity descriptions?
Separately dereferenceable Activity description aren't required by the Activity Streams specification and seems just to create a multitude of small documents that are redundant because the information is already in the Change Set. 

I see that LC Change Sets do not include separately dereferenceable links for each activity (seem to include some bnode ids but that is just clutter in the serialization, not for dereferencing)

### PR DESCRIPTION
**Pull request by zimeon**
Fixes #102, quite a big change with section changes too

### CODE REVIEW

File: docs/0.1/index.md [hunk 1]
```diff
@@ -449,45 +434,48 @@ Reference: [prev](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-prev) pr
...
+Not all implementations will store every change for an entity over time. A _Collection_{:.term} _MAY_{:.strong-term} provide feeds of only the last known metadata update for each entity. In the case where the _Collection_ provides feeds of only the last known metadata update for each entity case, the page identifier cannot be used to know the last _Activities_{:.term} processed by a consumer. For this reason the _Activities_{:.term} within the _Collection_{:.term} _MUST_ have a date property and _SHOULD_{:.strong-term} include the date using the `published` property. The `updated` property _SHOULD_ be used on the _Object_{:.term} for when the entity actually occurred. This level is sufficient to address the [Entity Change Activities List](#entity-change-activities-list) use case.
 
-This level is sufficient to address the [Entity Change Activities List](#entity-change-activities-list) use case.
+_Entity Change Activity_{:.term} objects appear in the `orderedItems` array within a [Change Set](#change-set) response.
 
-#### FULL EXAMPLE for Entity Change Activity:
+#### Example except for an Entity Change Activity
```
Comment (sfolsom) [inline docs/0.1/index.md:472]: excerpt?

File: docs/0.1/index.md [hunk 2]
```diff
@@ -449,45 +434,48 @@ Reference: [prev](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-prev) pr
 
 A link to the previous _Change Set_{:.term} in this _Entry Point_{:.term} for the _Entity Collection_{:.term}.
 
-The _Change Set_{:.term} _MAY_{:.strong-term} have a _prev_{:.term} property if there are
+The _Change Set_{:.term} c property if there are
```
Comment (sfolsom) [inline docs/0.1/index.md:437]: Should this be:

The _Change Set_{:.term} _MAY_{:.strong-term} have a `prev` property if there are

**Review by sfolsom — COMMENTED** [review]
@zimeon thanks for this major pull request. I see a couple typos, but other than that, it looks good.

File: docs/0.1/index.md (hunk 2 above)
Comment (zimeon) [reply docs/0.1/index.md:437]: Yes, missed that one, will fix

File: docs/0.1/index.md (hunk 1 above)
Comment (zimeon) [reply docs/0.1/index.md:472]: Indeed, thanks

**Review by sfolsom — APPROVED** [review]

**Pull request merged**